models.py

```diff
@@ -97,37 +103,44 @@ def build_cnn_model(sequence_length,
     input_shape = (sequence_length, vocab_size)
     dropout_prob = 0.25
 
-    # 32 filters, kernel width 5
-    model.add(
-        tf.keras.layers.Conv1D(
-            32, 5, activation='relu', input_shape=input_shape, padding='same'))
-    model.add(tf.keras.layers.Conv1D(32, 5, activation='relu', padding='same'))
-    model.add(tf.keras.layers.Conv1D(32, 5, activation='relu', padding='same'))
+    model.add(tf.keras.layers.Conv1D(num_filters, kernel_size, activation='relu', input_shape=input_shape, padding='same'))
+    model.add(tf.keras.layers.Conv1D(num_filters, kernel_size, activation='relu', padding='same'))
+    model.add(tf.keras.layers.Conv1D(num_filters, kernel_size, activation='relu', padding='same'))
     model.add(tf.keras.layers.Flatten())
-    model.add(tf.keras.layers.Dense(64, activation='relu'))
+    model.add(tf.keras.layers.Dense(hidden_size, activation='relu'))
     model.add(tf.keras.layers.Dropout(dropout_prob, seed=0))
     model.add(tf.keras.layers.Dense(1))
     model.summary()
 
-    adam_learning_rate = 0.0001
     model.compile(
         optimizer=tf.keras.optimizers.Adam(learning_rate=adam_learning_rate),
         loss='mse',
         metrics=['mse'])
     return model
 
 
-def get_model(model_name, sequence_length,
-              vocab_size):
+def get_model(model_name,
+              sequence_length: int,
+              vocab_size: int,
+              ridge_alpha: float=1.0,
```

Comment:
I would have found this more readable and extensible if it took 'model_kwargs: dict' as an arg and then the dict was unpacked inside build_linear_model and build_cnn helpers. Right now, it's awkward to add new hparams or new models bc you have to surface their kwargs all the way to get_model.

Reply:
I agree and separated this out into helpers. I think I like having the model-specific kwarg names have the model name prepended, and I don't mind pulling them out explicitly `num_filters=kwargs['cnn_num_filters'],` in the helper, even though it means I can't just pass `**kwargs` around like a magician